### PR TITLE
Hong Tat, Wendy, Azizi, Sujono, Siwat Kaoleung, Zac

### PR DESCRIPTION
**Pull request by hongtat88**

### CODE REVIEW

File: projects/site2019/src/assets/data/speakers.json [hunk 1]
```diff
@@ -297,5 +297,116 @@
...
+            "type": "github"
+        }],
+        "favourite": {
+            "icon": "",
+            "name": "Maggie Goreng Tambah Telur Mata"
+        }
+    },
+    {
+        "id": "zac",
+        "name": "Zac",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: Zachary Ng

File: projects/site2019/src/assets/data/speakers.json [hunk 2]
```diff
@@ -297,5 +297,116 @@
...
+            "type": "facebook"
+        }],
+        "favourite": {
+            "icon": "",
+            "name": "Maggie Goreng Tambah Telur Mata"
+        }
+    },
+    {
+        "id": "siwat-kaolueng",
+        "name": "JaMe",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: Siwat Kaolueng

File: projects/site2019/src/assets/data/speakers.json [hunk 3]
```diff
@@ -297,5 +297,116 @@
             "icon": "",
             "name": "Maggie Goreng Tambah Telur Mata"
         }
+    },
+    {
+        "id": "hongtat-yew",
+        "name": "HongTat Yew",
+        "title": "Senior Software Engineer @ ViewPoint Software for Business",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: Senior Software Engineer @ ViewPoint

File: projects/site2019/src/assets/data/speakers.json [hunk 4]
```diff
@@ -297,5 +297,116 @@
...
+        }],
+        "favourite": {
+            "icon": "",
+            "name": "Maggie Goreng Tambah Telur Mata"
+        }
+    },
+    {
+        "id": "wendy-kong",
+        "name": "Wendy Kong",
+        "title": "Software Engineer @ Pulsifi",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: Women Who Code KL Lead, Software Engineer

File: projects/site2019/src/assets/data/speakers.json [hunk 5]
```diff
@@ -297,5 +297,116 @@
...
+        }],
+        "favourite": {
+            "icon": "",
+            "name": "Maggie Goreng Tambah Telur Mata"
+        }
+    },
+    {
+        "id": "azizi-yazit",
+        "name": "Azizi Yazit",
+        "title": "Senior Web Developer @ Esri Malaysia",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: Senior Web Developer @ Esri

File: projects/site2019/src/assets/data/speakers.json [hunk 6]
```diff
@@ -297,5 +297,116 @@
...
+        }],
+        "favourite": {
+            "icon": "",
+            "name": "Maggie Goreng Tambah Telur Mata"
+        }
+    },
+    {
+        "id": "siwat-kaolueng",
+        "name": "JaMe",
+        "title": "Angular Developer @ 7 Peaks Software",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: Developer @ 7 Peaks Software

File: projects/site2019/src/assets/data/speakers.json [hunk 7]
```diff
@@ -297,5 +297,116 @@
...
+        }],
+        "favourite": {
+            "icon": "",
+            "name": "Maggie Goreng Tambah Telur Mata"
+        }
+    },
+    {
+        "id": "sujono",
+        "name": "Sujono",
+        "title": "Senior Front End Engineer @ Hong Leong Bank Berhad",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: Senior Front End Engineer @ Hong Leong Bank

File: projects/site2019/src/assets/data/speakers.json [hunk 8]
```diff
@@ -297,5 +297,116 @@
...
+            "url": "https://github.com/perjerz3434",
+            "type": "github"
+        }],
+        "favourite": {
+            "icon": "",
+            "name": "Maggie Goreng Tambah Telur Mata"
+        }
+    },
+    {
+        "id": "zac",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: consistent, change it to zachary-ng

File: projects/site2019/src/assets/data/speakers.json [hunk 9]
```diff
@@ -297,5 +297,116 @@
             "icon": "",
             "name": "Maggie Goreng Tambah Telur Mata"
         }
+    },
+    {
+        "id": "hongtat-yew",
+        "name": "HongTat Yew",
```
Comment (chybie) [inline projects/site2019/src/assets/data/speakers.json]: is your name really together or separate? 
- It's ok If you like to put it this way, or
- Hong Tat, Yew
- Hong Tat
- Yew Hong Tat

File: projects/site2019/src/assets/data/speakers.json (hunk 1 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 2 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 3 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 5 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 4 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 6 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 7 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 8 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: done

File: projects/site2019/src/assets/data/speakers.json (hunk 9 above)
Comment (hongtat88) [reply projects/site2019/src/assets/data/speakers.json]: it's actually yew hong tat. but people tend to call me hong, yew or yew tat. so i will make it as a word and they know hongtat is first name and yew is last name.

**Pull request merged**